packages/cli/generators/relation/index.js

```diff
@@ -374,6 +380,25 @@ module.exports = class RelationGenerator extends ArtifactGenerator {
     });
   }
 
+  async promptRegisterInclusionResolver() {
+    // should we let users decide the relation is traversable?
+    // or promt a warning msg: 'Notice not all models are traversable'
+    return this.prompt([
+      {
+        type: 'confirm',
+        name: 'isInclusionResolverRegistered',
```

Comment:
I find this name misleading - it says whether the resolver is already registered. Shouldn't it say something along the lines that the resolver will be registered?

```suggestion
        name: 'registerInclusionResolver',
```